### DISCUSSION TITLE
Proposal: add shadow-outline-gray, shadow-outline-blue, etc.

### DISCUSSION THREAD

**Discussion by bipinrajbhar** [tailwindlabs/tailwindcss#1730] [Ideas]
Hey :wave: , @adamwathan and @reinink 

It would be great if we have

shadow-outline-red
shadow-outline-green
shadow-outline-blue
etc.

**Comment by heikkilamarko** [comment]
The upcoming [Tailwind UI](https://tailwindui.com/) implements these utilities (colored focus shadows) as can be seen [here](https://tailwindui.com/documentation#how-tailwindcss-ui-extends-tailwind).

Hopefully these will be added to the Tailwind CSS framework as well.
